src/services/litegraphService.ts

```diff
@@ -948,6 +949,20 @@ export const useLitegraphService = () => {
     options: Record<string, any> = {}
   ): LGraphNode {
     options.pos ??= getCanvasCenter()
+    if (nodeDef.name.startsWith('SubgraphBlueprint')) {
+      const canvas = canvasStore.getCanvas()
+      const bp = useSubgraphStore().getBlueprint(nodeDef.name)
+      const items = { nodes: bp.nodes, subgraphs: bp.definitions?.subgraphs }
+      //FIXME
+      const strippedItems = JSON.parse(JSON.stringify(items))
+      const results = canvas._deserializeItems(strippedItems, {
+        position: options.pos
+      })
+      if (!results) throw new Error('Failed to add subgraph blueprint')
+      const node = results.nodes.values().next().value
+      if (!node) throw new Error('Failed to add subgraph blueprint')
```

Comment:
It might be helpful to differentiate between the ways this is failing, if a consumer only looks at the message and not the stacktrace.